apps/marketplace/components/SellerSelect/SellerSelect.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import AUtextInput from '@gov.au/text-inputs/lib/js/react.js'
+import AUbutton from '@gov.au/buttons/lib/js/react.js'
+import findSuppliers from 'marketplace/actions/supplierActions'
+import styles from './SellerSelect.scss'
+
+const SellerSelectView = props => (
+  <div>
+    <label htmlFor={props.id}>Select sellers</label>
+    <AUtextInput
+      id={props.id}
+      placeholder={props.placeholder}
+      onChange={props.handleOnChange}
+      value={props.inputValue}
+      className={props.className}
+    />
+    {props.showSearchButton && <AUbutton onClick={props.handleSearchClick}>Search</AUbutton>}
+  </div>
+)
+
+const SellerSelectResultsView = props => (
```

Comment:
In this instance separating the components into view/UI and data components is probably not useful but I think it's a pattern that could be very useful going forward for components that could be used in multiple places.